web/Controllers/HomeController.cs

```diff
@@ -27,7 +27,7 @@ public IActionResult Index()
 
             if (user.LandlordId != null)
             {
-                return RedirectToAction("MyDashboard", "Landlord");
+                return RedirectToAction("Dashboard", "Landlord");

```

Comment:
One thing to try and would make sure all of these are for correct methods is name the actions as nameof(LandlordController.Dashboard) etc :)

Reply:
Oh yeah that's a good idea!